MANIFEST.in

```diff
@@ -3,3 +3,5 @@ include *.txt
 recursive-include cf_units *.py
 include cf_units/etc/site.cfg.template
 include CHANGES COPYING COPYING.LESSER INSTALL
+include versioneer.py
+include cf_units/_version.py
```

Comment:
> Isn't cf_units/_version.py already included by the earlier recursive-include?

I confess, I let versioneer do this for me automatically. You are right of course.

> Do we need setup.cfg listed in the manifest as well?

I don't believe so, in the same way we do not need ``setup.py``.